src/docs/product/relay/options.mdx

```diff
@@ -246,223 +73,51 @@ human-readable strings of a number and a human-readable unit, such as:
+| metrics.buffering    | Boolean                   | default: `true`         | Whether the emitted metrics will be buffered before sending to StatsD server. This generally improves performance, but comes with a caveat that with low traffic volume, metrics might take a few more seconds to propagate.                                                                                                                                                                                                                                        |
+| metrics.sample_rate  | Float                     | default: `1.0`          | Global sample rate for all emitted metrics. Should be between 0.0 and 1.0. If outside that range, the value will be normalized (negative values would become 0.0, positive larger than 1.0 become 1.0). For instance, the value of 0.3 means that only 30% of the emitted metrics will be sent. Note that the implemented sampling approach is not aware of metrics types, and, for example, does not rescale counter values when the sample rate is less than 1.0. |
 
 ## Internal Error Reporting
 
 Configures error reporting for errors happening within Relay. Disabled by
 default.
 
-`sentry.enabled`
-
-: _boolean, default: `false`_
-
-Whether to report internal errors to a separate DSN. `false` means no internal
-errors are sent, but still logged.
-
-`sentry.dsn`
-
-: _String, optional_
-
-Sentry DSN to which to report internal Relay failures.
-
-We recommend setting this to a value that will not send Relay errors to
-itself. Ideally, this value should send errors to Sentry directly, not another
-Relay.
+| key            | type    | value            | description                                                                                                                                                                                                           |
+|----------------|---------|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| sentry.enabled | Boolean | default: `false` | Whether to report internal errors to a separate DSN. `false` means no internal errors are sent, but still logged.                                                                                                     |
+| sentry.dsn     | String  | optional: `true` | Sentry DSN to which to report internal Relay failures.  We recommend setting this to a value that will not send Relay errors to itself. Ideally, this value should send errors to Sentry directly, not another Relay. |
```

Comment:
```suggestion
| sentry.enabled | Boolean | default: `false` | Determines whether to report internal errors to a separate DSN. `false` means that no internal errors will be sent, but will still be logged.                                                                                                     |
| sentry.dsn     | String  | optional: `true` | Sentry DSN to which you can report internal Relay failures.  We recommend setting this to a value that sends Relay errors directly to Sentry instead of sending them to itself first. |
```